include/linux/bpf.h

```diff
@@ -92,7 +92,8 @@ typedef struct
     uint32_t key_size;          ///< Size in bytes of keys.
     uint32_t value_size;        ///< Size in bytes of values.
     uint32_t max_entries;       ///< Maximum number of entries in the map.
-    uint32_t map_flags;         ///< Flags (currently 0).
+    uint32_t map_flags;         ///< Not supported, must be zero.
```

Comment:
Could you please clarify the reason for changing the comment - ///< Flags (currently 0).

Reply:
I changed it to match the other fields that have to be zero. Happy to change this back.

Reply:
I think the changed comment is clearer the way Lorenz has it.